### DISCUSSION TITLE
This client isn't compatible with the media and the server isn't sending a compatible media format.

### DISCUSSION THREAD

**Discussion by JobberRT** [jellyfin/jellyfin-web#5249] [Q&A] [answered]
## First of all
Sorry for another post about this error, but I came into it and have searched the github and the forum, found no solution for it.

## System Information
CPU: 12400
RAM: 4GB
DISK: 64GB
Running in PVE VM with SR-IOV vGPU passthrough. Validated vGPU is working by playing other media file

## FFmpeg Log
[Log.txt](https://github.com/jellyfin/jellyfin-web/files/14470019/Log.txt)

## Jellyfin HWA Setting Screenshot
**Only the part that I changed and sorry for not changing it to English, I tried in the webui and force clear the cache, but it didn't help**

![1](https://github.com/jellyfin/jellyfin-web/assets/17498767/f84f24e6-5cce-46dd-b06b-0598b4771470)
![2](https://github.com/jellyfin/jellyfin-web/assets/17498767/6b879c10-9e02-41e5-aee2-366698d325f6)

### I don't know what problem here I have, but I can play some other movies normally. I'm software engineer so I can provide detailed information and I want to, please tell me if you need anything else

**Comment by thornbill** [comment]
Duplicate of https://github.com/jellyfin/jellyfin-web/discussions/3410

**Reply by JobberRT** [reply]
Yes, I saw that discussion, but that discussion is marked as answered with no solution. So I start another one here.

Additionally, I found this issue, will try after work to see if it's 7.1 audio problem(looks like the same as mine problem): https://github.com/jellyfin/jellyfin-web/issues/5039

**Comment by JobberRT** [comment, marked as answer]
Yes, this is indeed a audio channels problem, will continue discussion there(#5039 ), closing.
